index.js

```diff
@@ -119,7 +124,7 @@ class TagInput<T> extends React.PureComponent<Props<T>, State> {
   props: Props<T>;
   state: State = {
     text: '',
-    inputWidth: 90,
+    inputWidth: this.props.inputDefaultWidth,
```

Comment:
Does this actually work? I always assumed these declarations ran before the constructor. Even if they don’t, it’s probably cleaner to define a constructor that sets the state

Reply:
...the final result works, just tried to break in constructor after super(props), it could already log this.state.inputWidth which is initialised by my custom prop of 200. 
But you are right, I've move the initialisation of state to constructor, thanks~